client/src/pages/GroupExpenses/Components/GroupExpenseTable.tsx

```diff
@@ -0,0 +1,195 @@
+import * as React from 'react';
+import { Checkbox, Stack, Typography } from '@mui/material';
+import { GridColDef, GridRenderCellParams } from '@mui/x-data-grid';
+
+import {
+  splitDataWithId,
+  GroupExpenseTableProps,
+  GroupExpenseAsPaid,
+  GroupExpenseData,
+  GroupExpenseDataWithId,
+} from '../../../interfaces/interfaces';
+import { BackgroundBox, ExpensesDataGrid } from './styles';
+import { isoToFormattedDate } from '../../../utils/utils';
+import { categories } from '../../../constants/constants';
+import { theme } from '../../../assets/styles';
+/**
+ * Expenses Group table component
+ *
+ * @param {ExpenseData[]} expenses - expenses to display
+ * @returns {JSX.Element} - Group expenses table
+ */
+export const GroupExpensesTable: React.FC<GroupExpenseTableProps> = ({
+  groupExpenses,
+  handleMarkAsPaidGroupExpense,
+  currUserId,
+}) => {
```

Comment:
JSDoc comment is inaccurate

Reply:
yeah I'll clean them for stuff I touched in this pr, i'll clean the rest of the client on a different pr :)